ip/draft-ietf-detnet-ip.xml

```diff
@@ -569,7 +578,7 @@ DetNet                    |L2/SbN|          |L2/SbN|
           but that have not been the subject of a completed reservation,
           can disrupt the QoS offered to properly reserved DetNet flows
           by using resources allocated to the reserved flows.
-          Therefore, the network nodes of a DetNet network must:
+          Therefore, the network nodes of a DetNet network MUST:
 
```

Comment:
I think there's an open issue on this text that I thought I fixed a while back -- will submit under a separate PR

Reply:
OK.